apps/slackbot/bolt_app.py

```diff
@@ -54,8 +57,9 @@ def contains_verbosex(query: str) -> bool:
     return "-verbosex" in query.lower()
 
 
-def get_response(question, previous_messages):
-    llm = ChatOpenAI(openai_api_key=cfg.OPENAI_API_KEY, request_timeout=120 , temperature=0)
+
+def get_response(question, previous_messages, user_id, team_id):
+    llm = SherpaChatOpenAI(openai_api_key=cfg.OPENAI_API_KEY, request_timeout=120 , user_id=user_id , team_id=team_id ,temperature=0)
```

Comment:
We should make the temperature a configurable option, such as an environment variable? Rather than hard-code it to 0. Maybe this can be addressed in a separate PR